src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './initializer'
+export * from './minimal-live-connect'
+export * from './standard-live-connect'
+
+export * as eventBus from './events/event-bus'
+export * as types from './utils/types'
```

Comment:
These two do not end-up in the resulting `index.js` and `index.mjs`.

Reply:
Whatever was going on with my build environment. After editing the file, it picked it up.